grunt/tasks/release.js

```diff
@@ -111,13 +111,15 @@ function bower() {
 function docs() {
   var done = this.async();
 
+  grunt.file.copy('build/react-' + VERSION + '.zip', 'docs/downloads/react-' + VERSION + '.zip');
+  grunt.file.copy('build/react.js', 'docs/js/react.js');
```

Comment:
My guess/understanding is that this gets run from the -stable branch then cherry-picked to master so should work correctly? At least all the .zip files end up on master somehow:

https://github.com/facebook/react/tree/master/docs/downloads